### PR TITLE
fix false handle for describe index response(#25363)

### PR DESCRIPTION
**Pull request by MrPresent-Han**
related: #25363

### CODE REVIEW

**Comment by MrPresent-Han** [comment]
rerun ut

**Comment by MrPresent-Han** [comment]
rerun ut

**Comment by MrPresent-Han** [comment]
rerun ut

**Comment by MrPresent-Han** [comment]
rerun ut

**Comment by MrPresent-Han** [comment]
rerun ut

File: internal/querycoordv2/meta/coordinator_broker.go [hunk 1]
```diff
@@ -248,15 +248,15 @@ func (broker *CoordinatorBroker) DescribeIndex(ctx context.Context, collectionID
 	ctx, cancel := context.WithTimeout(ctx, paramtable.Get().QueryCoordCfg.BrokerTimeout.GetAsDuration(time.Millisecond))
 	defer cancel()
 
-	resp, err := broker.dataCoord.DescribeIndex(ctx, &indexpb.DescribeIndexRequest{
+	resp, _ := broker.dataCoord.DescribeIndex(ctx, &indexpb.DescribeIndexRequest{
```
Comment (congqixia) [inline internal/querycoordv2/meta/coordinator_broker.go:251]: We could not ignore grpc error here

File: internal/querycoordv2/job/utils.go [hunk 2]
```diff
@@ -78,20 +78,15 @@ func loadPartitions(ctx context.Context,
 			return err
 		}
 	}
-	indexes, err := broker.DescribeIndex(ctx, collection)
-	if err != nil {
-		return err
-	}
```
Comment (congqixia) [inline internal/querycoordv2/job/utils.go:84]: we could not simply remove describe index here
index shall exist before loading
any suggestion here @yah01 @weiliu1031

**Review by congqixia — CHANGES_REQUESTED** [review]

**Pull request closed without merging**